docs/manual/java/guide/broker/KafkaClient.md

```diff
@@ -4,7 +4,23 @@ Lagom provides an implementation of the Message Broker API that uses Kafka. In t
 
 ## Dependency
 
-To use this feature add the following in your project's build.
+You may need to use Kafka to publish data and/or you may need to subscribe to a topic providing data from third parties. When your service acts only as a Topic Subscriber you can import the Lagom Kafka Client alone.
```

Comment:
I had not though of 'A service that consumes a topic produced by a non-Lagom service' explicitly. I'll see how to connect it to http://www.lagomframework.com/documentation/1.2.x/java/ServiceClients.html